//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// acdef_T_V.h
//
// Code generation for function 'acdef_T_V'
//

#ifndef ACDEF_T_V_H
#define ACDEF_T_V_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acdef_T_V(double V_init, double A_init, double V_wayp, double V_max,
               double A_min, double J_max, double J_min, double T,
               creal_T t[14]);

#endif
// End of code generation (acdef_T_V.h)
